include/console/console.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_CONSOLE_H_
```

Comment:
Same broken header guard updates in this patch as previous ones.

If we're going to go this route, I think there should be sanitycheck test cases that use the old style, so we know they still work. As said before, this is going to be really important if anyone wants to maintain an application that's compatible with LTS and this new hierarchy.